wrap-tool.Rmd

```diff
@@ -4,5 +4,25 @@ Led by: Allie Cliffe, Broad Institute of MIT and Harvard
 
 AnVIL Outreach coordinator: Javier Carpinteyro-Ponce
 
+## About
 
-Workflow Description Language (WDL) not in your programming skillset? No problem! Learn to wrap your pipelining tool in WDL and run it in an AnVIL workspace in this CoFest session. We’ll go over WDL basics and how to run pipelines in AnVIL, then walk through how to run your own Unix/Python/R script. Along the way, you’ll write a WDL and build a custom Docker to run it with. Come and wrap a tool of interest! All experience levels are welcome. If time allows, we’ll emerge with some resources for others to run their non-WDL tool in AnVIL. 
+Workflow Description Language (WDL) not in your programming skillset? No problem! Learn to wrap your pipelining tool in WDL and run it in an AnVIL workspace in this CoFest! session. We’ll go over WDL basics and how to run pipelines in AnVIL, then walk through how to run your own Unix/Python/R script. Along the way, you’ll write a WDL and build a custom Docker to run it with. Come and wrap a tool of interest! All experience levels are welcome. We’ll brainstorm and, if time allows, we’ll create resources (tutorials? AnVIL book? Cheat sheet?) for others to use to run their non-WDL tool in AnVIL.  
```

Comment:
Maybe we could add links? e.g., `[Docker](https://www.docker.com/)`

Reply:
I think that'd be helpful. These are the ones I can think of:

- [WDL](https://openwdl.org/)
- [Docker](https://www.docker.com/)
- [AnVIL](https://anvilproject.org/)

Would that be enough?

Reply:
I'll merge this, and I could modify the blurb with the links in a later PR :)